base/util.go

```diff
@@ -279,3 +279,9 @@ func SplitTokens(cmd string) (tokens []string, userErrorMessage string, err erro
 		return nil, "", fmt.Errorf("error splitting command string: %s", err)
 	}
 }
+
+func IsDirectPrivateMessage(msg chat1.MsgSummary) bool {
+	return msg.Channel.MembersType != "team" &&
+		(msg.Sender.Username == msg.Channel.Name ||
+			len(strings.Split(msg.Channel.Name, ",")) == 2)
```

Comment:
This will cause it to skip sending the authorize message to restricted bots in 2 person impteams afaik, not sure we want that?

Reply:
we also invert this call, i think the logic is preserved as it was